CHANGELOG.md

```diff
@@ -2448,6 +2448,10 @@ If there are any bugs, improvements, optimizations or any new feature proposal f
 
 ### Added
 
+#### web3
+
+-   Updated type `Web3EthInterface.accounts` to includes `privateKeyToAccount`,`privateKeyToAddress`,and `privateKeyToPublicKey` (#6762)
```

Comment:
Sorry, is this the correct change for this MR?

Reply:
It was `chacngelog sync` fix